src/core/currency/wallet/currency-wallet-cleaners.ts

```diff
@@ -246,7 +254,37 @@ export const asEdgeTxActionStake = asObject({
   stakeAssets: asArray(asEdgeAssetAmount)
 })
 
-export const asEdgeTxAction = asEither(asEdgeTxActionSwap, asEdgeTxActionStake)
+export const asEdgeTxActionFiatType: Cleaner<EdgeTxActionFiatType> = asValue(
+  'buy',
+  'sell',
+  'sellNetworkFee'
+)
+
+export const asEdgeTxActionFiat = asObject<EdgeTxActionFiat>({
+  type: asEdgeTxActionFiatType,
+
+  orderId: asString,
+  walletId: asString,
```

Comment:
Why is there a walletId in here? We are already attached to an `EdgeTransaction`, and that has a `walletId` on it.

Reply:
That's fair. I'll remove.